core/events/tests/test_api.py

```diff
@@ -105,20 +105,6 @@ def test_save_from_scratch_without_creation_event(self, mock_database):
         with self.assertRaises(NoSuchSubmission):
             save(e2)
 
-    @mock.patch('events.classic')
```

Comment:
This was no longer relevant after the changes to events.save (InvalidEvent percolates up).